### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by GuardianLinkDev**

### CODE REVIEW

File: src/adapters/glexchange-polygon-1/index.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "glexchange-polygon-1";
+        this.protocol = "polygon";
+	this.token = "0xc2132d05d31c914a87c6611c10748aeb04b58e8f";
+        this.block =  34825626;
+        this.contract = "0x8013bf7e0278891FB82b26FbD56628e12FEd112A";
```
Comment (liutaurasjanuskas) [inline src/adapters/glexchange-polygon-1/index.ts:31]: All EVM addresses has to be lowercased.

File: src/adapters/glexchange-polygon-1/index.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    contract: string;
+    events: string[];
+    pathToAbi: string;
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "glexchange-polygon-1";
+        this.protocol = "polygon";
```
Comment (liutaurasjanuskas) [inline src/adapters/glexchange-polygon-1/index.ts:28]: Update protocol to `matic`. We don't support `polygon` yet.

File: src/adapters/glexchange-polygon-1/index.ts (hunk 1 above)
Comment (GuardianLinkDev) [reply src/adapters/glexchange-polygon-1/index.ts]: Lowercased here https://github.com/GuardianLinkDev/nft-sales-adapters/blob/main/src/adapters/glexchange-polygon-1/index.ts#L69

Anyway I have changed from uppercase to lowercase

File: src/adapters/glexchange-polygon-1/index.ts (hunk 2 above)
Comment (GuardianLinkDev) [reply src/adapters/glexchange-polygon-1/index.ts]: Changed protocol to matic . but in your sdk you have mentioned as polygon  here https://github.com/dappradar/nft-sales-adapters/blob/main/src/sdk/constants.ts#L4

**Pull request merged**